modules/programs/rofi/default.nix

```diff
@@ -3,6 +3,8 @@
 with lib;
 let
   cfg = config.soxin.programs.rofi;
+
+  inherit (soxin.packages.${pkgs.system}) rofi-i3-support;
```

Comment:
Soxin's overlay is added to nixpkgs, we should then use `pkgs` here, so if the user overrides it, it gets passed here.

Reply:
That's not the case any longer. As you can see [here](https://github.com/SoxinOS/soxin/blob/personal/soxin/move-to-flakeutils/overlays/default.nix), the overlay is now empty. Should we add the overlay? I'm concerned if the overlay conflicted with another, what do you think of adding a top-level namespace (such as pkgs.soxin.<something>)?

Reply:
Do those evaluate properly with flakes?

Reply:
I just posted an update, can you check if you like it?

Reply:
It looks good!